modules/geo-layers/src/index.ts

```diff
@@ -31,6 +31,7 @@ export {default as TerrainLayer} from './terrain-layer/terrain-layer';
 export {default as MVTLayer} from './mvt-layer/mvt-layer';
 
 export {getURLFromTemplate as _getURLFromTemplate} from './tile-layer/utils';
+export {default as _Tileset2D} from './tile-layer/tileset-2d';
```

Comment:
Should we document in the `tile-layer.md` doc that this is currently exported with an underscore?

Reply:
Add to the code example